### DISCUSSION TITLE
[Protokoll] 12.3.2024

### DISCUSSION THREAD

**Discussion by gabrieltorresgamez** [AdversarialAttacks/.github#18] [Protokolle]
### Wer hat an dieser Sitzung teilgenommen?

- [X] Gabriel Torres Gamez
- [X] Si Ben Tran
- [X] Daniel Perruchoud
- [X] Stephan Heule

### Welche sind die wichtigsten Themen, die an dieser Sitzung besprochen wurden?

An dieser Sitzung haben wir unsere neuste Idee und das Feedback von Dani und Stephan besprochen. Weiterhin haben wir einige Details zu der Vereinbarung geklärt.

### Was wurde an der Sitzung beschlossen?

- Wir werden Anhand vom Feedback unsere Projektvereinbarung finalisieren.
- Ein möglichst finaler Prototyp der Vereinbarung werden wir bis Freitag einreichen.
- Beim Evaluationsprotokoll in der Vereinbarung müssen wir passende Metriken für unsere Datensätze definieren.
- Risiken sollten wir einschätzen (Eintrittswahrscheinlichkeit/Schadenshöhe)
- Dani/Stephan werden uns über Overleaf Feedback geben (@7ben18: Zugriff gewährleisten)

**Comment by gabrieltorresgamez** [comment]
Protokoll BAT 12.03.24

Ben hat Fachexperten über das Mail von Roswitha informiert.
Stephan findet, dass die Transferierbarkeit von Adversarial PEtrubationen dazu führen könnte, dass Ensembles nicht allzu gut wirken, wir können das jedoch analysieren
Bei Ablation Study findet Stephan, dass wir verschiedene Arten von Attacken anschauen können (FGSM,PGD,etc.). Analyse: Sind einige Perfomanter als andere? Verschiedene Attacken analysieren wären interessant.
Ablation Study wird nicht gemeint, dass man Bestandteile vom Modell entfernt.
Ben hat Fachexperten über unsere Datensätze gezeigt.
Frage: Haben die Bilder alle nur eine Farbdimension?
Wir analysieren zuerst einen Datensatz, danach je nach Ressourcen können wir die Übertragbarkeit analysieren.
Ben hat erklärt, was wir mit spezifischen Pertubationen (Abschnitt 5) meinen.
Bei der Erklärbarkeit/Interpretierbarkeit hat Ben erklärt, dass wir Saliency Maps nutzen wollen, um zu schauen wo ein attackiertes Modell hinschaut.
Bei der Analyse nach Adversarial Training würden wir neue adversarial Attacken generieren, und danach mit diesen Attacken die Robustheit des Modells testen.
Wär noch interessant, die Reihenfolge der Bilder bei der Generation von universal Pertrubationen zu shuffeln, um viele universelle Pertubationen zu generieren.
Wir müssen uns Gedanken über die Evaluation machen (evaluation auf true labels oder unterschiede unsecured/secured)
Dani/Stephan Zugriff senden für Overleaf
Vorschlag bis Freitag senden!
3.3 drinnen lassen, 3.4 brauchts nicht.
Evaluationsprotokoll ist wichtig. (Welche Metriken nehmen wir (soll Modellspezifisch sein))
Risiken sollten wir Einschätzen (Wahrscheinlichkeit/Problemwertung)

**Comment by 7ben18** [comment]
> Dani/Stephan werden uns über Overleaf Feedback geben (@7ben18: Zugriff gewährleisten)

Einladung via Overleaf für [report](https://www.overleaf.com/project/65c62e3ffdad90aaed4fcc6e) und [project-agreement](https://www.overleaf.com/project/65d4657d59691a194b165709) gesendet. 

Siehe unter Overleaf report oder project-agreemnt -> Oben rechts auf Share. 

![image](https://github.com/AdversarialAttacks/.github/assets/66916399/44e7c77a-fd53-4e88-a276-f389b29b2afc)

PS: Falls aus irgendeinem Grund es nicht funktionieren sollte, sind auf Teams die Share Links für View Ansicht und Edit Ansicht hinterlegt.
